release/release.go

```diff
@@ -37,41 +37,33 @@ type OS struct {
 	VersionID string `json:"version-id,omitempty"`
 }
 
+var (
+	versionSignaturePath = "/proc/version_signature"
```

Comment:
This is specifically to see if we're on ubuntu with !ubuntu kernel. We don't want to degrade to !apparmor in such cases.